x-pack/plugins/infra/public/components/feature_feedback_button.tsx

```diff
@@ -5,46 +5,77 @@
  * 2.0.
  */
 
+import React, { ReactElement } from 'react';
 import { EuiButton } from '@elastic/eui';
-import React from 'react';
 import { FormattedMessage } from '@kbn/i18n-react';
 import { useKibanaContextForPlugin } from '../hooks/use_kibana';
 
 const KIBANA_VERSION_QUERY_PARAM = 'entry.548460210';
+const KIBANA_DEPLOYMENT_TYPE_PARAM = 'entry.573002982';
 
-const getSurveyFeedbackURL = (formUrl: string, kibanaVersion?: string) => {
+const getDeploymentType = (
+  isCloudEnabled?: boolean,
+  isServerlessEnabled?: boolean
```

Comment:
I believe they can be non-optional as defined in the `InfraClientStartDeps` interface

Reply:
True, updated